repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/serializer/GoogleSpreadSheetUtils.java

```diff
@@ -0,0 +1,35 @@
+package fr.inria.spirals.repairnator.serializer;
+
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.model.AppendValuesResponse;
+import com.google.api.services.sheets.v4.model.ValueRange;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Created by fernanda on 22/03/17.
+ */
+public class GoogleSpreadSheetUtils {
```

Comment:
hmmm instead of creating a new class, can't you put that in GoogleSpreadsheetFactory?